### ISSUE TITLE
Application displaying black screen before running

### ISSUE THREAD

**Issue by MohammadeMBi** [flutter/flutter#49932]
hello,
when I open my application displaying a black screen after a white screen and then displaying my application, How can I solve this problem?

**Comment by bero93** [comment]
i think its just the time it takes to load fluttter activity , you can add splash screen to android to show a logo or something while the app is loading

**Comment by MohammadeMBi** [comment]
> 
> 
> i think its just the time it takes to load fluttter activity , you can add splash screen to android to show a logo or something while the app is loading

problem not resolved

**Comment by VladyslavBondarenko** [comment]
Hi @MohammadeMBi 
I see open issues addressing the case you described.
Please follow up on https://github.com/flutter/flutter/issues/47582 or https://github.com/flutter/flutter/issues/20827
I'm closing the current one as duplicate.
If you disagree please provide in comments your `flutter doctor -v` and explain what is different in your case, and I will reopen it.
Thank you

**Comment by praveenb** [comment]
Even i created a new Flutter project, I see app showing white screen, next black screen and then loads Flutter Home Page. Tested on master and stable channel same issue.

I able to fix white screen, by creating splash screen. but not black screen.

Finally, I found a solution doing as below
![image](https://user-images.githubusercontent.com/592596/73973538-c2080580-4948-11ea-9832-892ea07da9ec.png)

If i **uncheck**,` Include Kotlin Support for Android code`, app doesn't show blank screen now.
Not really sure about exact reason for it.

Hope it helps somebody.
